aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -265,23 +250,22 @@ impl VMChangeSet {
         let events = events.into_iter().map(|event| (event, None)).collect();
         let change_set = Self {
             resource_write_set,
-            module_write_set,
             delayed_field_change_set: BTreeMap::new(),
             aggregator_v1_write_set: BTreeMap::new(),
             aggregator_v1_delta_set: BTreeMap::new(),
             events,
         };
-        checker
-            .check_change_set(&change_set)
-            .map_err(|e| e.finish(Location::Undefined))?;
-        Ok(change_set)
+        (change_set, module_write_set)
     }
 
     /// Converts VM-native change set into its storage representation with fully
     /// serialized changes. The conversion fails if:
     /// - deltas are not materialized.
     /// - resource group writes are not (combined &) converted to resource writes.
-    pub fn try_into_storage_change_set(self) -> Result<StorageChangeSet, PanicError> {
+    pub fn try_into_storage_change_set(
```

Comment:
nit: `try_combine_into_storage_change_set`, we are not just converting the VMChangeSet